### PR TITLE
first commit for team section

### PR DESCRIPTION
**Pull request by FernandaFT**
Review section 'Nuestro Equipo'
Review url /que-es-bae
can you see in styleguide
screen shot 2017-12-02 at 11 19 15 am

### CODE REVIEW

**Comment by betovarg** [comment]
update

File: grav/user/themes/azul/blueprints/about.yaml [hunk 1]
```diff
@@ -123,6 +123,9 @@ form:
                 .member_name:
                   type: text
                   label: Nombre del integrante
+                .category_name:
```
Comment (betovarg) [inline grav/user/themes/azul/blueprints/about.yaml]: Corregir este texto.

**Comment by FernandaFT** [comment]
the team section should look like this
screen shot 2017-11-25 at 5 03 01 pm

**Comment by betovarg** [comment]
1. Description needs an update example:

> - Log into this url [url], should look like: [screenshot]
> - Go into the styleguide in this url [url], should look like this: [screenshot]

2. Images are missing
3. background is missing both in the about page and in the styleguide
4. this is what I see:
styleguide:
image

about page:
image

**Comment by FernandaFT** [comment]
Images update

**Comment by betovarg** [comment]
@FernandaFT images are still missing:

These are the changes in your commit:
```
 grav/user/themes/azul/src/img/ArcelioChavez.jpg     | Bin 0 -> 13009 bytes
 grav/user/themes/azul/src/img/CarlosFallas.jpg      | Bin 0 -> 17200 bytes
 grav/user/themes/azul/src/img/DarnerMora.jpg        | Bin 0 -> 18190 bytes
 grav/user/themes/azul/src/img/FloraCubero.png       | Bin 0 -> 19727 bytes
 grav/user/themes/azul/src/img/ManriqueHernandez.jpg | Bin 0 -> 14037 bytes
 grav/user/themes/azul/src/img/MarianaFonseca.jpg    | Bin 0 -> 11500 bytes
 grav/user/themes/azul/src/img/MarlenCruz.jpg        | Bin 0 -> 17863 bytes
 grav/user/themes/azul/src/img/Rectangle.jpg         | Bin 0 -> 1024 bytes
 grav/user/themes/azul/src/img/RobertoAzofeifa.jpg   | Bin 0 -> 16797 bytes
 grav/user/themes/azul/src/img/SergioBolanos.jpg     | Bin 0 -> 14219 bytes
 11 files changed, 71 insertions(+)
 create mode 100644 grav/user/themes/azul/img/pattern-olas.svg
 create mode 100644 grav/user/themes/azul/src/img/ArcelioChavez.jpg
 create mode 100644 grav/user/themes/azul/src/img/CarlosFallas.jpg
 create mode 100644 grav/user/themes/azul/src/img/DarnerMora.jpg
 create mode 100644 grav/user/themes/azul/src/img/FloraCubero.png
 create mode 100644 grav/user/themes/azul/src/img/ManriqueHernandez.jpg
 create mode 100644 grav/user/themes/azul/src/img/MarianaFonseca.jpg
 create mode 100644 grav/user/themes/azul/src/img/MarlenCruz.jpg
 create mode 100644 grav/user/themes/azul/src/img/Rectangle.jpg
 create mode 100644 grav/user/themes/azul/src/img/RobertoAzofeifa.jpg
 create mode 100644 grav/user/themes/azul/src/img/SergioBolanos.jpg
```

This is what the browser is looking for:
image

**Comment by betovarg** [comment]
@FernandaFT FINALLY.

Wont pass it until responsive is done.

**Comment by FernandaFT** [comment]
the team section should look like this in responsive
screen shot 2017-12-02 at 1 18 06 pm

**Comment by betovarg** [comment]
My changes for rwd:

image
image
image

**Comment by betovarg** [comment]
🚢

**Pull request merged**